Dockerfile

```diff
@@ -19,17 +19,20 @@ ARG SPARK_IMAGE=gcr.io/spark-operator/spark:v2.4.0
 FROM golang:1.12.5-alpine as builder
 ARG DEP_VERSION="0.5.3"
 RUN apk add --no-cache bash git
-ADD https://github.com/golang/dep/releases/download/v${DEP_VERSION}/dep-linux-amd64 /usr/bin/dep
-RUN chmod +x /usr/bin/dep
-
+ENV GO111MODULE=on
 WORKDIR ${GOPATH}/src/github.com/GoogleCloudPlatform/spark-on-k8s-operator
-COPY Gopkg.toml Gopkg.lock ./
-RUN dep ensure -vendor-only
+COPY go.mod go.sum ./
+RUN go mod download
```

Comment:
followed this nice trick: https://container-solutions.com/faster-builds-in-docker-with-go-1-11/